samples/js-coffee-shop/package.json

```diff
@@ -5,7 +5,8 @@
   "main": "lib/index.js",
   "scripts": {
     "start": "node lib/index.js",
-    "genkit:dev": "genkit start -- tsx --watch src/index.ts",
+    "genkit:dev": "npx genkit start -- tsx --watch src/index.ts",
+    "setup": "npm i",
```

Comment:
unnecessary, remove

Reply:
Some of the samples need a setup step, so I wanted to have one in each for consistency

Reply:
ahh gotcha, sg